src/modules/AutoInstallRules.rb

```diff
@@ -36,6 +36,14 @@ def main
 
       Yast.include self, "autoinstall/io.rb"
 
+      reset
+      AutoInstallRules()
+    end
+
+    # Reset the module's state
```

Comment:
maybe related to comment above, reset to state to what? initial one? but then why not call Rules method?